app/models/asset_manager.rb

```diff
@@ -34,18 +34,13 @@ class AssetManager < ApplicationRecord
   validates_format_of :email, :with => /\A([^@\s]+)@((?:[-a-z0-9]+\.)+[a-z]{2,})\Z/i, :on => :create
   validates :phone, :presence => true, :numericality => true, :length => { :minimum => 10, :maximum => 15}
 
-  def read_answer(answer)
-    contains = false
-    if portfolio.read_answer(answer)
-      contains = true
-    end
-    contains
-  end
-
   def read_question(question)
     contains = false
-    if portfolio.read_question(question)
-      contains = true
+    answers.each do |answer|
```

Comment:
asset managers get read access to all questions for all building types they have access to